app/assets/javascripts/nested_related_items.js

```diff
@@ -0,0 +1,63 @@
+/* global Blacklight */
+/* global NestedRelatedItems */
+
+(function (global) {
+  var NestedRelatedItems;
+
+  NestedRelatedItems = {
+    options: { itemSelector: 'li.mods_display_nested_related_item' },
+    init: function (el) {
+      var _this = this;
+
+      _this.listItems(el).each(function() {
+        $(this).removeClass('open');
+        _this.addToggleLink($(this));
+      });
+    },
+
+    listItems: function(list) {
+      return list.find(this.options.itemSelector);
+    },
+
+    addToggleLink: function(content) {
+      var dl = content.find('dl');
+
+      if(dl.length > 0) {
+        // Hide and remove the dl (so we can deal with just the text below)
+        dl.hide();
+        dl.remove();
```

Comment:
Have you considered using  a `.detach` here? https://api.jquery.com/detach/

> The .detach() method is the same as .remove(), except that .detach() keeps all jQuery data associated with the removed elements. This method is useful when removed elements are to be reinserted into the DOM at a later time.

I'm not sure if we need to keep that info around or not.

Reply:
🤔 I'm not sure any jQuery will be associated with the `dl`, but happy to use `.detach` instead just in case.

Reply:
/shrug 👍